[libpopt] libpopt servers appear to be gone

**Host Environment**
- OS: Any
- Compiler: Any

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install libpopt`

**Failure logs** 
```
PS D:\vcpkg-master> .\vcpkg.exe install libpopt
Computing installation plan...
The following packages will be built and installed:
    libpopt[core]:x86-windows
Starting package 1/1: libpopt:x86-windows
Building package libpopt[core]:x86-windows...
-- Downloading http://rpm5.org/files/popt/popt-1.16.tar.gz...
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:99 (message):


  File does not have expected hash:

          File path: [ F:/downloads/temp/popt-1.16.tar.gz ]
      Expected hash: [ bae2dd4e5d682ef023fdc77ae60c4aad01a3a576d45af9d78d22490c11e410e60edda37ede171920746d4ae0d5de3c060d15cecfd41ba75b727a811be828d694 ]
        Actual hash: [ e9c99dcefad469c75cd4eb80eacff55f5d88fb6500309bc2a520d1d03b8aff2b00406de4fda2a47cae71847aa54ad214a1c5c5690a940badf723777d32a488a3 ]

  The file may have been corrupted in transit.  This can be caused by
  proxies.  If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY
  environment variables to
  "https://user:password@your-proxy-ip-address:port/".



Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:182 (test_hash)
  ports/libpopt/portfile.cmake:6 (vcpkg_download_distfile)
  scripts/ports.cmake:90 (include)


Error: Building package libpopt:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libpopt:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
```

**Additional context**
There are 2 servers for this package listed in the port file. One server, rpm5.org, appears to be completely gone. The other server, ftp.openbsd.org, appears to no longer host any version of this package.

We may want to consider outright delisting the package.

--- a/ports/libpopt/CONTROL
+++ b/ports/libpopt/CONTROL
@@ -1,3 +1,3 @@
 Source: libpopt
-Version: 1.16-12
+Version: 1.16-13
 Description: Library for parsing command line parameters

--- a/ports/libpopt/portfile.cmake
+++ b/ports/libpopt/portfile.cmake
@@ -1,11 +1,8 @@
-include(vcpkg_common_functions)
-
 set(LIBPOPT_VERSION 1.16)
 set(LIBPOPT_HASH bae2dd4e5d682ef023fdc77ae60c4aad01a3a576d45af9d78d22490c11e410e60edda37ede171920746d4ae0d5de3c060d15cecfd41ba75b727a811be828d694)
 
 vcpkg_download_distfile(ARCHIVE
     URLS
-        "http://rpm5.org/files/popt/popt-${LIBPOPT_VERSION}.tar.gz"
         "https://ftp.openbsd.org/pub/OpenBSD/distfiles/popt-${LIBPOPT_VERSION}.tar.gz"
     FILENAME "popt-${LIBPOPT_VERSION}.tar.gz"
     SHA512 ${LIBPOPT_HASH}
@@ -33,5 +30,4 @@ vcpkg_copy_pdbs()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
-file(COPY ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/libpopt)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/libpopt/COPYING ${CURRENT_PACKAGES_DIR}/share/libpopt/copyright)
+file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

